korea_news_crawler/sportcrawler.py

```diff
@@ -72,35 +81,30 @@ def make_sport_page_url(self, url, start_year, last_year, start_month, last_mont
 
             for month in range(target_start_month, target_last_month + 1):
                 for day in range(1, calendar.monthrange(year, month)[1] + 1):
-                    url = url
+                    url = input_url
                     if len(str(month)) == 1:
                         month = "0" + str(month)
                     if len(str(day)) == 1:
                         day = "0" + str(day)
                     url = f'{url}{year}{month}{day}'
                     # page 날짜 정보만 있고 page 정보가 없는 url 임시 저장
-                    final_url = url
+                    final_url = copy.deepcopy(url)
```

Comment:
```suggestion
                    final_url = f'${input_url}{year}{month}{day}'
```
이렇게 하고 84번, 89번 라인 url 변수 선언한 내용을 지웁니다.